app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -48,11 +49,13 @@ function PasswordConfirmStep({ errors, registerField, onChange, value }: Passwor
           )}
         </Alert>
       )}
-      {errors.map(({ error }) => (
-        <Alert key={error.message} type="error" className="margin-bottom-4">
-          {error.message}
-        </Alert>
-      ))}
+      {errors
+        .filter(({ error }) => error instanceof PasswordSubmitError)
```

Comment:
do non-password submit errors get rendered by a different section? or just omitted

Reply:
The only other errors we should be expecting are the field-level errors for the password input, which are rendered as an error message accompanying the field (0:02 in original comment video).